File: PHP/TimeConversion.php
<?php
/**
 * @version PHP 7.3.29
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
namespace PHP;

class TimeConversion {

    /*
     * Complete the 'timeConversion' function below.
     *
     * The function accepts INTEGER_ARRAY arr as parameter.
     */
    function timeConversion($s) {
        return date("H:i:s", strtotime($s));
    }

    /*
    $fptr = fopen(getenv("OUTPUT_PATH"), "w");
    $s = rtrim(fgets(STDIN), "\r\n");
    $result = timeConversion($s);
    fwrite($fptr, $result . "\n");
    fclose($fptr);
    */
}

File: GO/TimeConversion.go
/**
 * @version GO 1.19.2
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package GO

import (
	"time"
)

/*
 * Complete the 'timeConversion' function below.
 *
 * The function is expected to return a STRING.
 * The function accepts STRING s as parameter.
 */

func timeConversion(s string) string {
	tm, _ := time.Parse("03:04:05PM", s)

	return tm.Format("15:04:05")
}

/*
func main() {
    reader := bufio.NewReaderSize(os.Stdin, 16 * 1024 * 1024)
    stdout, err := os.Create(os.Getenv("OUTPUT_PATH"))
    checkError(err)

    defer stdout.Close()
    writer := bufio.NewWriterSize(stdout, 16 * 1024 * 1024)
    s := readLine(reader)

    result := timeConversion(s)
    fmt.Fprintf(writer, "%s\n", result)
    writer.Flush()
}

func readLine(reader *bufio.Reader) string {
    str, _, err := reader.ReadLine()
    if err == io.EOF {
        return ""
    }

    return strings.TrimRight(string(str), "\r\n")
}

func checkError(err error) {
    if err != nil {
        panic(err)
    }
}
*/
